doozerlib/distgit.py

```diff
@@ -1409,6 +1417,12 @@ def update_distgit_dir(self, version, release, prev_release=None):
             with io.open('Dockerfile', 'w', encoding="utf-8") as df:
                 for comment in oit_comments:
                     df.write("%s %s\n" % (OIT_COMMENT_PREFIX, comment))
+
+                # specify new env vars in the dockerfile
+                if env_vars:
+                    env_line = "ENV " + " ".join([key + "=" + val for key, val in env_vars.items()])
+                    df.write(f'{env_line}\n')
```

Comment:
i'm missing something about how this is being done right now (something is injecting env vars, i WROTE it, just don't see it at the moment) but this does not seem right. you would have to write it at the beginning of each stage, not just once.

Reply:
New push injects after each FROM